### ISSUE TITLE
Transformer block can cause NPE if the wire blocks aren't facing the correct direction

### ISSUE THREAD

**Issue by zachoooo** [ElectricBlocks/electricblocks#16]
```
[11:18:03] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: java.lang.NullPointerException
[11:18:03] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at edu.uidaho.electricblocks.simulation.SimulationConnection.toJson(SimulationConnection.java:37)
[11:18:03] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at edu.uidaho.electricblocks.simulation.SimulationHandler.simRequest(SimulationHandler.java:138)
[11:18:03] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at edu.uidaho.electricblocks.simulation.SimulationHandler.access$100(SimulationHandler.java:34)
[11:18:03] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at edu.uidaho.electricblocks.simulation.SimulationHandler$1.run(SimulationHandler.java:54)
```

**Comment by zachoooo** [comment]
After update error is now
```
Exception in thread "Thread-18" [11:48:00] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: java.lang.NullPointerException
[11:48:00] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at edu.uidaho.electricblocks.simulation.SimulationConnection.toJson(SimulationConnection.java:37)
[11:48:00] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at edu.uidaho.electricblocks.simulation.SimulationNetwork.toJson(SimulationNetwork.java:224)
[11:48:00] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at edu.uidaho.electricblocks.simulation.SimulationHandler.simRequest(SimulationHandler.java:127)
[11:48:00] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at edu.uidaho.electricblocks.simulation.SimulationHandler.access$100(SimulationHandler.java:34)
[11:48:00] [Thread-18/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: 	at edu.uidaho.electricblocks.simulation.SimulationHandler$1.run(SimulationHandler.java:54)
```

**Comment by zachoooo** [comment]
fixed in 48d0662
